templates/_helpers.tpl

```diff
@@ -29,4 +29,11 @@ Create imagepullsecretfor docker registry
 {{- with .Values.imageCredentials }}
 {{- printf "{\"auths\":{\"ghcr.io\":{\"username\":\"cobrowse-enterprise\",\"password\":\"%s\",\"auth\":\"%s\"}}}" .password (printf "cobrowse-enterprise:%s" .password | b64enc) | b64enc }}
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+Create the default SecurityContext for all service components
+*/}}
+{{- define "defaultSecurityContext" }}
+{{- dict "allowPrivilegeEscalation" false "privileged" false "readOnlyRootFilesystem" true | toYaml }}
```

Comment:
We also wanted to add `runAsNonRoot: true` here to signify that we are not running as root inside the container, but this requires us to change all our Dockerfiles to use `1000` as the user and not `"node"`.

I think you'd be in favour of that but thought I'd keep this PR smaller to focus to read only FS.